ClientDependency.Core/FileRegistration/Providers/BaseFileRegistrationProvider.cs

```diff
@@ -24,6 +24,7 @@ public abstract class BaseFileRegistrationProvider : ProviderBase
         protected BaseFileRegistrationProvider()
         {
             EnableCompositeFiles = true;
+            EnableCompositeSingleFiles = false;

```

Comment:
Maybe rename property name since files are not "composite" when in single files.